torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1483,7 +1478,9 @@ def _make_tensor_helper(shape, low=None, high=None):
            autodiff_nonfusible_nodes=['aten::add', 'aten::mm'],
            skips=(
                SkipInfo('TestCommon', 'test_variant_consistency_jit',
-                        dtypes=[torch.bfloat16, torch.float16, torch.cfloat, torch.cdouble]),),
+                        dtypes=[torch.bfloat16, torch.float16, torch.cfloat, torch.cdouble]),
+               # addmm does not correctly warn when resizing out= inputs
+               SkipInfo('TestCommon', 'test_out')),
```

Comment:
and yet addmm was one of the offenders on cross-device args not throwing proper runtime errors, so skipping whole test feels wrong. nbd, because those cases are separately tested, but wrong.

Reply:
We could make the test more granular. I hope we don't continue to see a hodgepodge of incorrectly implemented out= behavior, however. If the skips continue to increase then we should think about being more stringent or increasing test granularity.